### PR TITLE
Fix support for deeper structure in folder parameter

### PR DESCRIPTION
**Pull request by dariosaltarelli**
Fix logic when using subfolder paths in "folder" input (i.e "docs/MySubsection"). This allow to use the action multiple time with different parent id pages (usefull when separating content into different pages on Confluence).

Exiting action does not support it as it builds an incorrect path (root+path) when generating markdown which doubles the subfolder elements. So for example given:

docs/MySubsection/my.md

it will try to read:

docs/MySubsection/MySubsection/my.md

### CODE REVIEW

**Review by Bhacaz — COMMENTED** [review]
Hi, thanks a lot for the fix! 

Could you run the build command and commit the result please.

```shell
npm run build
```

**Comment by dariosaltarelli** [comment]
You mean like this? Screenshot 2024-12-02 at 18 40 47

**Comment by Bhacaz** [comment]
> You mean like this? Screenshot 2024-12-02 at 18 40 47

Yes.

File: dist/index.js
Comment (Bhacaz) [inline dist/index.js:1]: I don't know what append, but it doesn't seems right. 

I very sorry for the back and fork, I should do something to handle the build step. It should not be on the contributers responsibility.

To facilitate this PR I would like to propose to you, to simply revert the changes in the `dist` folder and I will handle the build once merge.

Is that ok with you?

Comment (dariosaltarelli) [reply dist/index.js:1]: Reverted last commit. Let me know if this is now ok.

**Pull request merged**